### PR TITLE
Tidy up examples

### PR DESCRIPTION
**Pull request by rparrett**

### CODE REVIEW

File: examples/3d/blend_modes.rs [hunk 1]
```diff
@@ -188,7 +188,7 @@ fn setup(
     let label_text_style = TextStyle {
         font: asset_server.load("fonts/FiraMono-Medium.ttf"),
         font_size: 25.0,
-        color: Color::WHITE,
+        color: ORANGE.into(),
```
Comment (rparrett) [inline examples/3d/blend_modes.rs:191]: I think this particular text-change-to-white may have been unintentional.

File: examples/ecs/removal_detection.rs [hunk 2]
```diff
@@ -55,7 +55,7 @@ fn react_on_removal(mut removed: RemovedComponents<MyComponent>, mut query: Quer
     // `Component` `T` (in this case `MyComponent`) removed at some point earlier during the frame.
     for entity in removed.read() {
         if let Ok(mut sprite) = query.get_mut(entity) {
-            sprite.color.set_alpha(0.5);
+            sprite.color = Color::srgb(0.5, 1., 1.);
```
Comment (rparrett) [inline examples/ecs/removal_detection.rs:58]: From testing, the 50% alpha just didn't seem like an obvious enough change visually.

This should be identical to what it did before.

**Comment by alice-i-cecile** [comment]
Thank you very much: I really appreciate it :) Reading over these examples for a dozen hours has definitely reduced my sensitivity to this sort of thing.

File: examples/ecs/state.rs [hunk 3]
```diff
@@ -82,7 +82,7 @@ fn setup_menu(mut commands: Commands) {
                         "Play",
                         TextStyle {
                             font_size: 40.0,
-                            color: Color::WHITE,
+                            color: Color::srgb(0.9, 0.9, 0.9),
```
Comment (rparrett) [inline examples/ecs/state.rs:85]: I think this text-change-to-white may have been unintentional. Matching our other "button text grey."

**Pull request merged**